packages/block-library/src/audio/transforms.js

```diff
@@ -32,8 +32,10 @@ const transforms = {
 			attributes: {
 				src: {
 					type: 'string',
-					shortcode: ( { named: { src } } ) => {
-						return src;
+					shortcode: ( {
+						named: { mp3, m4a, ogg, wav, wma, src },
+					} ) => {
+						return mp3 || m4a || ogg || wav || wma || src;
```

Comment:
We should read attributes with the same preference as the shortcode itself:

https://github.com/WordPress/WordPress/blob/af523447f46c351cb27432813273049777fbb890/wp-includes/media.php#L2859-L2878

If `src` exists, it will get used, and only if not, individual extension attributes are checked. Let's move `src` to the begining of the `||` chain and we should match it.

Reply:
Great spot!

Reply:
Fixed.

Reply:
THANKS all!